transport/src/main/java/io/netty5/channel/DefaultChannelHandlerContext.java

```diff
@@ -880,8 +943,12 @@ private Future<Void> findAndInvokeSendOutboundEvent(Object event) {
     }
 
     private Future<Void> invokeSendOutboundEvent(Object event) {
+        Future<Void> failed = saveCurrentPendingBytesIfNeededOutbound();
+        if (failed != null) {
+            return failed;
```

Comment:
Don't we need to dispose of `event` here?

Reply:
good catch!